projects/examples/src/components/datagrid/datagrid.examples.module.ts

```diff
@@ -80,6 +82,11 @@ Documentation.registerDocumentationEntry({
             title: 'Selectable datagrid rows',
             urlSegment: 'datagrid-row-select',
         },
+        {
+            component: DatagridIsRowSelectableExampleComponent,
+            title: 'Enable or disable selection of datagrid rows',
```

Comment:
I thought we had agreed to "Disabling selection of datagrid rows"

Reply:
Updated the title